.lgtm.yml

```diff
@@ -0,0 +1,17 @@
+extraction:
+  cpp:
+    before_index:
+      # always use Python 3
+      - cp `which python3` $LGTM_WORKSPACE/python
+      - PATH=$LGTM_WORKSPACE:$PATH
+      # build protobuf from source with -fPIC
+      - git clone https://github.com/protocolbuffers/protobuf.git
+      - cd protobuf
+      - git checkout v3.16.0
```

Comment:
when we update protobuf to a next version we will need to update all these places... wondering whether there is an easier way to fetch the version from a common place... not implying that this needs to be done in this PR.

Reply:
Yes it is a burden when every time ONNX bumps protobuf version... Also I was considering to have a common script for building protobuf from source since we have quite many different CIs using the same commands.  I will add this investigation in my TODO list.